index.bs

```diff
@@ -885,7 +895,7 @@ spec:url; type:dfn; text:scheme
     The algorithm for determining the [[CSP#effective-directive-for-a-request|effective directive for a request]]
     is modified by prepending the following:
 
-    1. If |request|'s [=initiator=] is "", its [=destination=] is "`document`", and
+    1. If <var ignore>request</var>'s [=initiator=] is "", its [=destination=] is "`document`", and
```

Comment:
I'd leave this as it was, but change the previous line:

> The algorithm for determining the [[CSP#effective-directive-for-a-request|effective directive for a request]], |request| (a [=/request=]) is modified by prepending the following:

Reply:
Good idea. I've done this in https://github.com/WICG/portals/pull/180 and will rebase this on that when it's merged.